### ISSUE TITLE
1.12 images do not start

### ISSUE THREAD

**Issue by ml4** [canonical/multipass#3239]
**Describe the bug**
Images do not start.

**To Reproduce**
How, and what happened?
1. `multipass launch 22.04`
1. image creates immediately
1. image sits at starting... forever

**Expected behavior**
What did you expect to happen?
image to start

**Logs**
Please provide logs from the daemon, see [accessing logs](https://multipass.run/docs/accessing-logs) on where to find them on your platform.

**Additional info**
- OS: [e.g. macOS 10.15]
Ventura 13.4

- `multipass version`
$ multipass version
multipass   1.12.2+mac
multipassd  1.12.2+mac

- `multipass info --all`
Name:           intrepid-asp
State:          Unknown
IPv4:           --
Release:        --
Image hash:     bbf52c59f6a7 (Ubuntu 22.04 LTS)
CPU(s):         --
Load:           --
Disk usage:     --
Memory usage:   --
Mounts:         --

Name:           stable-parrotfish
State:          Stopped
IPv4:           --
Release:        --
Image hash:     bbf52c59f6a7 (Ubuntu 22.04 LTS)
CPU(s):         --
Load:           --
Disk usage:     --
Memory usage:   --
Mounts:         --

- `multipass get local.driver`
qemu

**Additional context**
I had it working a few months ago, but upgraded to 1.12 and stuff stopped working. At this point I preferred Parallels for other reasons, but prefer the lower weight of automating image deploys. I'm on M1.

$ multipass launch 22.04
launch failed: The following errors occurred:
intrepid-asp: timed out waiting for response

Logs:
[2023-09-26T18:46:49.116] [info] [intrepid-asp] process state changed to Running
[2023-09-26T18:46:49.116] [debug] [qemu-system-aarch64] [92376] started: qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:e9:56:b0 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/intrepid-asp/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/intrepid-asp/cloud-init-config.iso
[2023-09-26T18:46:49.116] [info] [intrepid-asp] process started
[2023-09-26T18:46:49.116] [debug] [intrepid-asp] Waiting for SSH to be up
[2023-09-26T18:46:49.245] [debug] [intrepid-asp] QMP: {"QMP": {"version": {"qemu": {"micro": 0, "minor": 0, "major": 8}, "package": ""}, "capabilities": ["oob"]}}

[2023-09-26T18:46:49.272] [debug] [intrepid-asp] QMP: {"return": {}}

[2023-09-26T18:47:02.622] [debug] [intrepid-asp] QMP: {"timestamp": {"seconds": 1695750422, "microseconds": 622858}, "event": "NIC_RX_FILTER_CHANGED", "data": {"path": "/machine/unattached/device[6]/virtio-backend"}}

<stuck at this point>

**Comment by sharder996** [comment]
Hi @ml4!

This is known issue that we've been seeing with Multipass on macOS which is coming from Apple's use of the firewall and network stack in macOS, and is largely out of our control. There are several other Github issues on this topic here already (#2387, #3220, as well as this [comment](https://github.com/canonical/multipass/issues/3003#issuecomment-1698855750)) that might provide some additional troubleshooting steps or workarounds. Let me know if any of them work for you!

**Comment by percygrunwald** [comment]
I was able to resolve this issue using the OnyX tool as recommended in another GH issue relating to VM start issues: https://github.com/canonical/multipass/issues/3003#issuecomment-1736353490

**Comment by ml4** [comment]
Hello thanks for your time on this.  Here's what I've done/found:

- Adding multipassd to the incoming allowed list on the M1 firewall - no
- Turning off the firewall completely - no
- Turn off the VPN - no
- Turn off the firewall and turning off the VPN - no
- Firewall on and ran these but no
/usr/libexec/ApplicationFirewall/socketfilterfw --add /usr/libexec/bootpd
/usr/libexec/ApplicationFirewall/socketfilterfw --unblock /usr/libexec/bootpd
- I saw a user updated to 13.5.1 but my M1 only offers me sonoma so I'm gonna upgrade to that now and see what happens.

**Comment by sharder996** [comment]
Hi @ml4, are you still experiencing this issue? Is the behavior/logs the same?

**Comment by townsend2010** [comment]
I'm going to close this as there has been no follow up for a while now.  If there is additional info, please reopen or open a new issue.  Thanks!
